specification/ai/HealthInsights/HealthInsights.RadiologyInsights/route.radiologyinsights.tsp

```diff
@@ -13,24 +12,15 @@ using TypeSpec.Http;
 namespace AzureHealthInsights;
 
 interface RadiologyInsights {
```

Comment:
Should this have the `radiology-insights` route?
 ```suggestion
@route("radiology-insights")
interface RadiologyInsights {
```

Reply:
ok. thanks